resources/views/login.blade.php

```diff
@@ -0,0 +1,55 @@
+<title>Login page</title>
```

Comment:
Please dont change the location of this file. It is supposed to remain in resources/views/auth/login.blade.php. It is from a package and it may bring errors in other places when the package tries to look for it and doesnt find it